src/oop/special_class_method/hard/q5/QuestionText.md

```diff
@@ -0,0 +1,43 @@
+# oop-special_class_method-hard-q5
+
+## Question Text
+
```

Comment:
much of this opening paragraph is made redundant by the description of the class below